### ISSUE TITLE
Iphone 9.4 ios 14.2 bootloop

### ISSUE THREAD

**Issue by sportusrcu** [checkra1n/BugTracker#1842]
**Tell us about your setup:**
1. What iDevice are you using?
Iphone 7 plus gsm
2. On what version of iOS is it?
Ios 14.2
3. What version of checkra1n are you using?
0.12.0
4. What is your host system (OS version? Hackintosh? VM? etc.)?
High Sierra 
5. How are you connecting to the device (USB-A? USB-C? Apple/3rd party cable? Through a USB hub?)?
Anker usb, like always.
**What are the steps to reproduce the issue?**
1. Normally just following the steps, everything worked but stuck on spinning wheel and apple logo forever.I’ve  been jailbroken since checkra1n released.
2. Waiting too boot and it finished successfully.
3. Spinning wheel appears and then apple logo, repeatedly 
...

**What do you expect, and what is happening instead?**
I expect to be jailbroken instead of bootloped.

**Does the issue also occur if you tick "Safe Mode" in the checkra1n options?**
Yes

**Any other info, error logs, screenshots, ...?**
I’ll try with another usb and update ticket.
No error appears
Edit:
Same thing, no go.
Restored and tried but same results.

EDIT:
basically don’t restore from icloud although i did restored rootfs, now jailbreake is successful and everything is working.
